src/components/CallScriptsCard.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <div class="scriptCordion">
```

Comment:
try to follow [kebab-case](https://en.wikipedia.org/wiki/Kebab_case) for css classes